push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/kafka/MessageHolderWithVariantsKafkaConsumer.java

```diff
@@ -9,36 +25,34 @@
 import javax.inject.Inject;
 
 /**
- * Created by matzew on 8/16/17.
+ * Consumes {@link MessageHolderWithVariants} objects from various topics
+ * and triggers {@link org.jboss.aerogear.unifiedpush.message.token.TokenLoader} for further
+ * processing
  */
 public class MessageHolderWithVariantsKafkaConsumer {
 
-
     private final Logger logger = LoggerFactory.getLogger(MessageHolderWithVariantsKafkaConsumer.class);
 
-    private final String ADM_TOPIC = "AdmMessage_Topic";
+    private final String ADM_TOPIC = "agpush_admPushMessageTopic";
 
-    private final String APNS_TOPIC = "APNsMessage_Topic";
+    private final String ANDROID_TOPIC = "agpush_gcmPushMessageTopic";
```

Comment:
@polinankoleva do you mind changing the topic name here too?

Reply:
No, change them everywhere. So that they can be the same.

Reply:
? I thought you had started the renaming process